Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2018"
 [features]
 # Enable this feature to turn on additional GC optimizations that are only
 # possible with the rustgc fork of the compiler.
-rustgc = []
+rustgc = ["libgc_internal/rustgc_internal"]
```

Comment:
Silly question: is the `libgc_internal/` bit here correct?

Reply:
Ouch. Good catch, thank you.